parlai/core/image_featurizers.py

```diff
@@ -139,6 +143,14 @@ def _image_mode_switcher(self):
             'resnet50_spatial': ['resnet50', -2],
             'resnet34_spatial': ['resnet34', -2],
             'resnet18_spatial': ['resnet18', -2],
+            'resnext101_32x8d_wsl': ['resnext101_32x8d_wsl', -1],
```

Comment:
Add a docstring to this function to explain that element 1 of each dictionary value is the cutoff layer idx?